packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -430,6 +472,12 @@ class LocalEngineArtifacts extends Artifacts {
         return fs.path.join(_hostEngineOutPath, _artifactToFileName(artifact));
       case Artifact.webPlatformKernelDill:
         return fs.path.join(_getFlutterWebSdkPath(), 'kernel', _artifactToFileName(artifact));
+      case Artifact.fuchsiaKernelCompiler:
+      case Artifact.fuchsiaPlatformDill:
+      case Artifact.fuchsiaPatchedSdk:
+      case Artifact.fuchsiaFlutterJitRunner:
+        assert(false, 'Invalid local engine artifact $artifact.');
```

Comment:
@iskakaushik Do you know where these would be in a local engine build?

Reply:
Sorry, missed this somehow. They would be in the `out/fuchsia_x64_.../flutter_aot_runner.far` for example.